core/src/main/groovy/noe/eap/utils/CLILib.groovy

```diff
@@ -2067,7 +2066,7 @@ class CLILib {
       boolean authJustAdded = false
       // Is realm's authentication undefined?
       if (CLILib.readArbitraryCommandOutput(securityRealmBuilder.as7serverInstance, cmdStrReadMgt) =~ /${securityRealmBuilder.name}[[^\n]*\n]*authentication[ "=>\{\n]*undefined/) {
-        if (StringUtils.isBlank(securityRealmBuilder.truststoreAbsolutePath) || StringUtils.isBlank(securityRealmBuilder.truststorePassword)) throw new IllegalArgumentException("One must set both TruststoreAbsolutePath and TruststorePassword.")
+        if (securityRealmBuilder.truststoreAbsolutePath?.allWhitespace || securityRealmBuilder.truststorePassword.isBlank()) throw new IllegalArgumentException("One must set both TruststoreAbsolutePath and TruststorePassword.")
```

Comment:
`isBlank()` for `securityRealmBuilder.truststorePasswor` is intentional here?

Reply:
Nice catch, I missed it in my last edit. Apparently the integration tests didn't catch it! I'm fixing it as I rebase.